Sofa/Component/Topology/Container/Grid/src/sofa/component/topology/container/grid/SparseGridTopology.cpp

```diff
@@ -182,6 +182,9 @@ void SparseGridTopology::init()
             _nodeCubesAdjacency[ hexahedra[i][j] ].push_back( i );
         }
     }
+
+	m_upperElementType = core::topology::TopologyElementType::HEXAHEDRON;
```

Comment:
```suggestion
    m_upperElementType = core::topology::TopologyElementType::HEXAHEDRON;
```

just a indent issue. Otherwise good to go. I'm surprised sparseGrid is not calling inherit::init()
If you could change your editor to use 4 spaces as Tab